src/core_atmosphere/Registry.xml

```diff
@@ -442,6 +442,8 @@
 			<var name="isltyp"/>
 			<var name="ivgtyp"/>
 			<var name="mminlu"/>
+			<var name="isice_lu"/>
+			<var name="iswater_lu"/>
```

Comment:
Are there additional changes needed in the `init_atmosphere` core to produce the `isice_lu` and `iswater_lu` fields?

Reply:
@mgduda 
I'm not sure about this. As I've modified it, everything compiles/runs okay, but maybe @ldfowler58 would have more insight?